tests/e2e/kserve_test.go

```diff
@@ -206,3 +210,60 @@ func (c *KserveTestCtx) validateDefaultCertsAvailable(t *testing.T) {
 	g.Expect(ctrlPlaneSecret.Type).Should(Equal(defaultIngressSecret.Type))
 	g.Expect(defaultIngressSecret.Data).Should(Equal(ctrlPlaneSecret.Data))
 }
+
+func (c *KserveTestCtx) ValidateKserveOwnsFeatureTrackers(t *testing.T) {
+	g := c.NewWithT(t)
+
+	fts := []string{
+		c.ApplicationNamespace + "-kserve-external-authz",
+		c.ApplicationNamespace + "-serverless-serving-gateways",
+		c.ApplicationNamespace + "-serverless-serving-deployment",
+		c.ApplicationNamespace + "-serverless-net-istio-secret-filtering",
+	}
+
+	for _, ft := range fts {
+		g.Get(
+			gvk.FeatureTracker, types.NamespacedName{Name: ft},
+		).Eventually().Should(
+			jq.Match(`.metadata.ownerReferences | any(.kind = "%s")`, c.GVK.Kind),
```

Comment:
I don't understand why, but the e2e test on the PR seems to be hitting a nil pointer dereference error at runtime from this line. I can't reproduce it locally for some reason, the test works fine.

@lburgazzoli is there anything obviously wrong with the jq here, or anything else you think might be causing it?

Reply:
probably == instead of = ?

Reply:
Urgh, that's definitely wrong. I've updated that now (and also changed the second argument to be gvk.Kserve.Kind to be clearer. Let's hope one or both of those fix it.

When I use `=` instead of `==` locally, I still get _a result_, even if it's not the correct result. I'm not sure if/how this could fix the nil pointer deref issue, but I'll cross my fingers! :crossed_fingers:

Reply:
Still the same issue even with the update. I'll try to dig deeper to see if I can understand what's going on

Reply:
It's due to  dereferencing of nil v in

``` 	case *unstructured.Unstructured:
		return v.Object, nil
	}
```
(jq_support.go:110) since Get (testf_witht.go:94) gets NotFound and returns nil, nil

Reply:
good catch, let me fix it

Reply:
@grdryn https://github.com/opendatahub-io/opendatahub-operator/pull/1698